v3/src/components/graph/models/graph-data-configuration-model.ts

```diff
@@ -397,27 +397,40 @@ export const GraphDataConfigurationModel = DataConfigurationModel
         }
       })
     },
-    binDetails() {
+    binDetails(_binAlignment?: number, _binWidth?: number, reset = false) {
       const kDefaultNumberOfBins = 4
-      const minValue = self.caseDataArray.reduce((min, aCaseData) => {
-        return Math.min(min, self.dataset?.getNumeric(aCaseData.caseID, self.primaryAttributeID) ?? min)
-      }, 0) ?? 0
-      const maxValue = self.caseDataArray.reduce((max, aCaseData) => {
-        return Math.max(max, self.dataset?.getNumeric(aCaseData.caseID, self.primaryAttributeID) ?? max)
-      }, 0) ?? 0
+
+      let minValue = NaN
+      for (const aCaseData of self.caseDataArray) {
+        const numericValue = self.dataset?.getNumeric(aCaseData.caseID, self.primaryAttributeID)
+        if (isFiniteNumber(numericValue)) {
+          minValue = isNaN(minValue) ? numericValue : Math.min(minValue, numericValue)
+        }
+      }
+      let maxValue = NaN
+      for (const aCaseData of self.caseDataArray) {
+        const numericValue = self.dataset?.getNumeric(aCaseData.caseID, self.primaryAttributeID)
+        if (isFiniteNumber(numericValue)) {
+          maxValue = isNaN(maxValue) ? numericValue : Math.max(maxValue, numericValue)
+        }
+      }
```

Comment:
As mentioned elsewhere, there was nothing wrong with the version that used `Infinity`. The only requirement is that none of these `Infinity` or `NaN` values gets propagated to the user.